core/trie/key.go

```diff
@@ -15,10 +15,12 @@ type Key struct {
 }
 
 func NewKey(length uint8, keyBytes []byte) Key {
+	// length should be less than 32 ? otherwise bytesNeeded() return wrong index
```

Comment:
length is in bits, so it can be as high as 251 in our usecase IIRC.

> otherwise bytesNeeded() return wrong index

why is that?